test/io/ferguson/buffer_test.chpl

```diff
@@ -1,35 +1,60 @@
 use Buffers;
 
+proc testassign() {
+  var b = new bytes(50);
+  {
+    // test assignment overload.
+    var b_tmp:bytes;
+    var b_tmp2 = new bytes();
+    b_tmp2 = b;
+    b_tmp = b;
+  }
+}
+
 proc main() {
+  testassign();
+
   // allocate 50 bytes
   var b = new bytes(50);
-
+  {
+    // test assignment overload.
+    var b_tmp:bytes;
+    var b_tmp2 = new bytes();
+    b_tmp2 = b;
+    b_tmp = b;
+  }
```

Comment:
This section looks like a repeat of testassign(), but I don't think an error that occurred there wouldn't occur here.  Should both be present?